src/apyfloatarray_wrapper.cc

```diff
@@ -652,6 +652,38 @@ void bind_float_array(nb::module_& m)
         APyFloatArray
             An array filled with the specified value.
     )pbdoc"
+        )
+        .def_static(
+            "arange",
+            &APyFloatArray::arange,
+            nb::arg("start"),
+            nb::arg("stop"),
+            nb::arg("step"),
+            nb::arg("exp_bits"),
+            nb::arg("man_bits"),
+            nb::arg("bias") = std::nullopt,
+            R"pbdoc(
+            Initializes an array filled with the specified value.
```

Comment:
* `arange(stop)`: Values are generated within the half-open interval [0, stop) (in other words, the interval including start but excluding stop).
*  `arange(start, stop)`: Values are generated within the half-open interval [start, stop).
*  `arange(start, stop, step)`: Values are generated within the half-open interval [start, stop), with spacing between values given by step.